scripts/usb_scan.py

```diff
@@ -233,7 +233,7 @@ def add_interface(self, interface):
         :return: None
         """
         if interface in self.interfaces:
-            log.debug("overriding existing interface: " + interface)
+            log.debug("overriding existing interface: " + str(interface))
```

Comment:
The change here is different from original in sematic, while I think this is correct and we should print this way: then the `debug_str` defined in the base class `UsbObject` can be used.